### PR TITLE
Update foundations image URLs to the latest release [REVPI-2776]

### PR DESCRIPTION
**Pull request by nbuchwitz**
In preparation of REVPI-2776

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,14 +7,14 @@ Works with both [Raspberry Pi OS](https://www.raspberrypi.org/software/operating
 
 *Raspberry Pi OS with desktop*
 ```
-curl -O https://downloads.raspberrypi.org/raspios_oldstable_armhf/images/raspios_oldstable_armhf-2022-01-28/2022-01-28-raspios-buster-armhf.zip
-unzip 2022-01-28-raspios-buster-armhf.zip
+curl -O https://downloads.raspberrypi.org/raspios_oldstable_armhf/images/raspios_oldstable_armhf-2022-09-26/2022-09-22-raspios-buster-armhf.img.xz
+xz -d 2022-09-06-raspios-buster-armhf.img.xz
```
Comment (zhan-kunbus) [inline README.md:11]: ***09-06***.img.xz ?

File: README.md [hunk 2]
```diff
@@ -7,14 +7,14 @@ Works with both [Raspberry Pi OS](https://www.raspberrypi.org/software/operating
...
 ```
 
 *Raspberry Pi OS Lite*
 ```
-curl -O https://downloads.raspberrypi.org/raspios_oldstable_lite_armhf/images/raspios_oldstable_lite_armhf-2022-01-28/2022-01-28-raspios-buster-armhf-lite.zip
-unzip 2022-01-28-raspios-buster-armhf-lite.zip
+curl -O https://downloads.raspberrypi.org/raspios_oldstable_lite_armhf/images/raspios_oldstable_lite_armhf-2022-09-26/2022-09-22-raspios-buster-armhf-lite.img.xz
+xz -d 2022-09-06-raspios-buster-armhf-lite.img.xz
```
Comment (zhan-kunbus) [inline README.md:17]: ...**09-06**...img.xz ?

File: README.md (hunk 1 above)
Comment (nbuchwitz) [reply README.md:11]: No: https://downloads.raspberrypi.org/raspios_oldstable_armhf/images/raspios_oldstable_armhf-2022-09-26/

File: README.md (hunk 2 above)
Comment (nbuchwitz) [reply README.md]: Good catch, thanks. I've updated the PR.

**Review by zhan-kunbus — APPROVED** [review]
LGTM

**Pull request merged**